IDEAS/Buildings/Components/Interfaces/ZoneInterface.mo

```diff
@@ -55,13 +64,24 @@ partial model ZoneInterface "Partial model for thermal building zones"
   Modelica.Blocks.Interfaces.RealInput yOcc if useOccNumInput
     "Control input for number of occupants, used by Occupants.Input and Occupants.AreaWeightedInput"
     annotation (Placement(transformation(extent={{140,20},{100,60}})));
+  Modelica.Blocks.Interfaces.RealInput mWat_flow(unit = "kg/s") 
+    if useWatFlowInput 
+    "Input for injecting water vapor into a zone" annotation(
+    Placement(visible = true, transformation(origin = {0, 50}, extent = {{140, 20}, {100, 60}}, rotation = 0), iconTransformation(origin = {0, 40}, extent = {{140, 20}, {100, 60}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput C_flow 
+    if useCFlowInput 
+    "Input for injecting CO2 into a zone" annotation(
+    Placement(visible = true, transformation(origin = {0, 70}, extent = {{140, 20}, {100, 60}}, rotation = 0), iconTransformation(origin = {0, 80}, extent = {{140, 20}, {100, 60}}, rotation = 0)));  
```

Comment:
This input connector appears outside of the icon. Can it be placed in a better way?
![image](https://github.com/open-ideas/IDEAS/assets/43753266/3023b318-7b94-49d1-9755-88cf858a48cc)

Reply:
Revised inputs as follows 
![image](https://github.com/open-ideas/IDEAS/assets/5970879/3106580a-7656-4dc5-8883-9fa3365ad670)

Reply:
Actually, OM rewrites the whole file when moving those icons. Can you perhaps do it such that that doesn't happen?

Reply:
We should be carefull that the input connectors do not coincide with the zone bus. Furthermore, the diagram position should be decoupled from the icon position. The original `Zone` icon looks like this:
![image](https://github.com/open-ideas/IDEAS/assets/43753266/90be63b0-9f30-46ae-a248-a3f2ef0b9f08)

**Proposal 1**
![image](https://github.com/open-ideas/IDEAS/assets/43753266/c294e119-d1ab-4664-b571-f02927bff329)
(Left (up-down): C_flow, mWat_flow, uLig; right (up-down): yOcc, phi, TSensor, ppm)

**Proposal 2**
![image](https://github.com/open-ideas/IDEAS/assets/43753266/f66b7c2b-f213-4eb6-a8b9-d7eaf093fd7a)
(Left (up-down): yOcc, C_flow, mWat_flow, uLig; right (up-down): phi, TSensor, ppm)

@Mathadon do you prefer one of both? Or a mix of both (something else)?

Reply:
I vote proposal 2. In use, that would mean that all schedules ect. are on the same side of the zone component probably resulting in cleaner models.

In extended IDEAS components that I made for my own use-cases, I have added something which is basicly an 'IAQ output/sensor bus'. Not for this PR, but maybe something to show you and have a chat about in the future. Too many connectors can become cumbersome quite fast.

Reply:
I prefer proposal 2 :) 

Another option is to do a complete overhaul: inputs left, outputs right, heat ports & propsbus at the bottom.. It would make sense to do this since we could make a new IDEAS release for the interzonal airflow stuff. Since reordering connectors will change all connection coordinates.. There are also still port_a and port_b as FluidPorts which are hidden and deprecated. These could be removed then.

Reply:
I agree with a revision of all connector locations. I propose to go for proposal 2 now, such that we limit the number of changes to those focussing on the actual new developments of this PR.
I'll create a new issue to change the connector locations for the upcoming release. @kldjonge we can include your input regarding the new buses in that issue.

Reply:
An issue is created here: #1335